frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -46,7 +46,7 @@ const LabelComponent = ({ clusterID, ...props }) => (
 );
 
 const SubTitleComponent = (props) => (
-  <ChartLabel {...props} style={{ fill: 'var(--pf-v6-chart-color-black-500)' }} />
+  <ChartLabel {...props} x={220} y={100} style={{ fill: 'var(--pf-v6-chart-color-black-500)' }} />
```

Comment:
`--pf-v6-chart-color-black-500` is not a valid PF5 var and needs updating.  See https://www.patternfly.org/tokens/all-patternfly-tokens/